tests/py/dynamo/conversion/harness.py

```diff
@@ -338,7 +338,7 @@ def run_test_with_dynamic_shape(
         input_specs,
         rtol=1e-03,
         atol=1e-03,
-        output_dtypes=None,
+        check_dtype=True,
         use_dynamo_tracer=False,
         enable_passes=False,
     ):
```

Comment:
Changed to get check_dtype arg like DispatchTestCase::run_test() to handle different output type.

Reply:
Can you also make the same changes for `run_test_compare_tensor_attributes_only` for consistency?